### ISSUE TITLE
ProxyConnectionError on python 3.8 (An invalid argument was supplied)

### ISSUE THREAD

**Issue by strayge** [romis2012/aiohttp-socks#12]
With python 3.8 got   
`
ProxyConnectionError: [Errno 10022] Can not connect to proxy <DOMAIN>:<PORT> [An invalid argument was supplied]`
on connect attempt.

Same code+proxy works well with python 3.7.

### OS
Windows 10.0.19041.1
### versions
aiohttp==3.6.2
aiohttp-socks==0.3.3

### Test code
```python
import aiohttp
from aiohttp_socks import ProxyType, ProxyConnector, ChainProxyConnector

async def fetch(url):
    connector = ProxyConnector.from_url('socks5://<USERNAME>:<PASSWORD>@<DOMAIN>:<PORT>')
    async with aiohttp.ClientSession(connector=connector) as session:
        async with session.get(url) as response:
            return await response.text()

await fetch('http://ya.ru/')
```

### Python 3.7.6
works as expected

### Python 3.8.1
```python
venv\Lib\site-packages\aiohttp\client.py in __aenter__(self)
   1011     async def __aenter__(self) -> _RetType:
-> 1012         self._resp = await self._coro
   1013         return self._resp
   1014

venv\Lib\site-packages\aiohttp\client.py in _request(self, method, str_or_url, params, data, json, cookies, headers, skip_auto_headers, auth, allow_redirects, max_redirects, compress, chunked, expect100, raise_for_status, read_until_eof, proxy, proxy_auth, timeout, verify_ssl, fingerprint, ssl_context, ssl, proxy_headers, trace_request_ctx)
    478                                          loop=self._loop):
    479                             assert self._connector is not None
--> 480                             conn = await self._connector.connect(
    481                                 req,
    482                                 traces=traces,

venv\Lib\site-packages\aiohttp\connector.py in connect(self, req, traces, timeout)
    521
    522             try:
--> 523                 proto = await self._create_connection(req, traces, timeout)
    524                 if self._closed:
    525                     proto.close()

venv\Lib\site-packages\aiohttp\connector.py in _create_connection(self, req, traces, timeout)
    856                 req, traces, timeout)
    857         else:
--> 858             _, proto = await self._create_direct_connection(
    859                 req, traces, timeout)
    860

venv\Lib\site-packages\aiohttp\connector.py in _create_direct_connection(self, req, traces, timeout, client_error)
    978
    979             try:
--> 980                 transp, proto = await self._wrap_create_connection(
    981                     self._factory, host, port, timeout=timeout,
    982                     ssl=sslcontext, family=hinfo['family'],

venv\Lib\site-packages\aiohttp_socks\connector.py in _wrap_create_connection(self, protocol_factory, host, port, **kwargs)
     99             rdns=self._rdns, family=self._proxy_family)
     24             self._create_socket()
---> 25             await self._connect_to_proxy()
     26         else:
     27             self._socket = _socket

venv\Lib\site-packages\aiohttp_socks\proxy\base_proxy.py in _connect_to_proxy(self)
     55         except OSError as e:
     56             self.close()
---> 57             raise ProxyConnectionError(
     58                 e.errno,
     59                 'Can not connect to proxy {}:{} [{}]'.format(

ProxyConnectionError: [Errno 10022] Can not connect to proxy <DOMAIN>:<PORT> [An invalid argument was supplied]
```

**Comment by strayge** [comment]
After changing <DOMAIN> to IP-address is started works.

**Comment by strayge** [comment]
It calls system call
```WSAStringToAddressW ( "<DOMAIN>", AF_INET, NULL, ... )```
which returns `10022` error code, as it does not expect domain name as first param.

Looks like somewhere skipped `getaddrinfo` call to resolve server name to ip address.

**Comment by strayge** [comment]
### system calls without proxy (success)
![image](https://user-images.githubusercontent.com/2664578/72120042-a02a5b80-3367-11ea-949e-a877ea6db210.png)
### system calls with proxy (error)
![image](https://user-images.githubusercontent.com/2664578/72120082-c05a1a80-3367-11ea-9aec-c0d0aa349930.png)

**Comment by romis2012** [comment]
Thank you for explanation

**Comment by romis2012** [comment]
We are aware of the problem and working on it

**Comment by romis2012** [comment]
Fixed in ver 0.3.4

**Comment by strayge** [comment]
Thanks for fast fix! Now works.
